provisioner/provisioner.go

```diff
@@ -153,6 +166,16 @@ func (p *freenasProvisioner) GetConfig(storageClassName string) (*freenasProvisi
 		case "targetGroupPortalgroup":
 			targetGroupPortalgroup, _ = strconv.Atoi(v)
 
+			// Authentication options
```

Comment:
This looks like the comment may have a tab in it? It's pushed over further than the rest of the code in the diff interface in github.

Reply:
fixed